bin/ansible-lint

```diff
@@ -73,6 +73,10 @@ def main(args):
                       default=hasattr(sys.stdout, 'isatty') and sys.stdout.isatty(),
                       action='store_false',
                       help="disable colored output")
+    parser.add_option('--force-color', dest='force_colored',
```

Comment:
Use `dest=colored` and `action=store_true`, then you don't need the check at lines 91-96.